edgelet/edgelet-docker/src/settings.rs

```diff
@@ -494,18 +466,18 @@ mod tests {
         );
     }
 
-    #[test]
-    fn agent_image_is_resolved() {
-        let _env_lock = ENV_LOCK.lock().expect("env lock poisoned");
-        std::env::set_var("AZIOT_EDGED_CONFIG", GOOD_SETTINGS_IMAGE_RESOLVE);
-        let mut settings = Settings::new().unwrap();
-        agent_image_resolve(&mut settings).unwrap();
+    // #[test]
```

Comment:
Can we relocate the test?

Reply:
And remove the commented statements above as well?

Reply:
so, the tricky thing with this test is that it relies on the late binding of the `parent_hostname` value, which isn't something that is easily mocked...

Given just how much of a bodge this functionality is now, I think I'm just going to remove the test entirely.